csharp/src/Drivers/Databricks/DatabricksConnection.cs

```diff
@@ -89,6 +110,35 @@ protected override TOpenSessionReq CreateSessionRequest()
             return req;
         }
 
+        protected override void ValidateOptions()
+        { 
+             base.ValidateOptions();
+
+            bool tempUnavailableRetryValue = true; // Default to enabled
+            // Parse retry configuration parameters
+            if(Properties.TryGetValue(DatabricksParameters.TemporarilyUnavailableRetry, out string? tempUnavailableRetryStr))
+            {
```

Comment:
I assume you'd intended to parse or otherwise analyze the string configuration value? Right now, `tempUnavailableRetryValue` is never set to anything other than its default value of `true` .

Reply:
Thanks for mollifying the linter! This is still an open item.